datalake/src/test/java/com/sequenceiq/datalake/flow/SdxReactorFlowManagerTest.java

```diff
@@ -245,6 +249,16 @@ public void testSdxUpgradeRequestBackupNotRequested() {
         verify(reactor, times(1)).notify(eq(DATALAKE_UPGRADE_EVENT.event()), any(Event.class));
     }
 
+    @Test
+    void testTriggerCcmUpgradeFlow() {
+        sdxCluster = getValidSdxCluster("7.2.10");
+        try {
+            ThreadBasedUserCrnProvider.doAs(USER_CRN, () -> underTest.triggerCcmUpgradeFlow(sdxCluster));
+        } catch (Exception ignored) {
+        }
```

Comment:
I wonder why the need for the `try {} catch`. `ThreadBasedUserCrnProvider.doAs()` does not throw any checked exceptions (and using this method would not be strictly necessary for an UT scenario), and any `RuntimeException`s should probably be considered a failure in `triggerCcmUpgradeFlow()`. I suppose this pattern was "borrowed" from `testSdxUpgradeRequestBackupNotRequested()`. :smile:

Reply:
since there is no real reactor behind and the flow is not accepted within 10 seconds there is a timeout exception thrown